libraries/Microsoft.Bot.StreamingExtensions/RequestEx.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    /// <summary>
+    /// Implementation split between Request and RequestEx.
+    /// The basic request type sent over Bot Framework Protocol 3 with Streaming Extensions transports,
+    /// equivalent to HTTP request messages.
+    /// </summary>
+    public partial class Request
+    {
+        /// <summary>
+        /// Verb used by requests to get resources hosted on a remote server.
+        /// </summary>
+        public const string GET = "GET";
+
```

Comment:
how is this different from the Http GET request? 

Isn't this awfully similar to:
[https://docs.microsoft.com/en-us/previous-versions/visualstudio/hh138202(v=vs.118](https://docs.microsoft.com/en-us/previous-versions/visualstudio/hh138202(v=vs.118))

Reply:
These requests do not formally adhere to HTTP, however the protocol mirrors HTTP as best it can with similar potential verbs. This is why we did not take formal dependencies on the HTTP definitions.